components/brave_wallet/browser/zcash/rust/librustzcash/BUILD.gn

```diff
@@ -56,6 +56,10 @@ rust_static_library("zcash_primitives") {
     "src/zcash_primitives/src/transaction/components/orchard.rs",
     "src/zcash_primitives/src/transaction/mod.rs",
   ]
+
+  # TODO(cypt4): Fix unused imports and remove this flag.
+
+  rustflags = [ "-Aunused-imports" ]
```

Comment:
we fixed in the previous commit but not here? Maybe it's best to add this to both and fix them in a follow-up?

Reply:
The issue with this lib is that it is a third party library, however it is not being pulled by `gnrt`. cc @cypt4

Reply:
Yes, it is added by DEPS since we remove unused modules. Seems further cleanup is needed. Here is an issue 
https://github.com/brave/brave-browser/issues/43418